travis/travis_after_tests_success

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -v
+
+# if the env variable TESTS is not set, default to 1 to keep backward compatibility
+: ${TESTS:="1"}
+
+if [ "${TESTS}" = "1" ] && [ "${LINT_CHECK}" != "1" ] ; then
+  coveralls
+fi
```

Comment:
In python this could be:

```python
#!/usr/bin/env python

import os
from coveralls import cli as coveralls_cli

if os.environ.get("TESTS") and not os.environ.get("LINT_CHECK"):
    coveralls_cli.main(argv=None)
```